Source/WebCore/dom/EventTarget.h

```diff
@@ -150,6 +150,21 @@ class EventTarget : public ScriptWrappable, public CanMakeWeakPtrWithBitField<Ev
         return nullptr;
     }
 
+    template <typename CallbackType>
+    void enumerateEventListenerTypes(CallbackType callback)
```

Comment:
Can this be `const`?

Reply:
Indeed fixed along side the space removal in the line above it.